website/content/en/status/report-2023-04-2023-06/pot.adoc

```diff
@@ -0,0 +1,26 @@
+=== Containers and FreeBSD: Pot, Potluck and Potman
+
+Links: +
+link:https://github.com/bsdpot[Pot organization on GitHub] URL: link:https://github.com/bsdpot[]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@freebsd.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@freebsd.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During the last quarter, link:https://github.com/bsdpot/pot/releases/tag/0.15.5[pot 0.15.5] was released, containing a number of bugfixes and attribute features from various contributors. It will be available on the 2023Q3 quarterly package set. 
```

Comment:
Can you clarify what you mean by "attribute features"?

Reply:
Added explanation and link.